<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp_server</class>
 <widget class="QDialog" name="tcp_server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcp_server_lv</string>
  </property>
  <widget class="QPushButton" name="button_print_to_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Print to box</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_stop_server">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop server</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start_server">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear box</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>button_print_to_box</tabstop>
  <tabstop>button_stop_server</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
